modules/Template/PortalPublication.yaml

```diff
@@ -0,0 +1,70 @@
+#######################################
+# Portal          
+#######################################
+
+# Note: this template definition uses the alternate syntax with 'attribute'
+
+classes:
+  PublicationTemplate:
+    is_a: Template
+    description: >
+      A generalized data model template for Publications to be hopefully used across dccs
+    notes:
+    - This structure is based on a CSV file from MC2 annotations with customized fields as attributes.
+    attributes:
+      grantNumber:
+        range: string
+        required: true
+      consortiumName:
+        range: string
+        required: true
+      themeName:
+        range: string
+        required: true
+      doi:
+        range: string
+        required: true
+      journal:
+        range: string
+        required: true
+      pmid:
+        range: integer
+        required: true
+      pubmedUrl:
+        range: string
+        required: true
+      title:
+        range: string
+        required: true
+      year:
+        range: integer
+        required: true
+      diseaseFocus:
```

Comment:
`diseaseFocus` should be moved to NF subclass.

Reply:
This is probably the NFType attribute I made, will replace with diseasefocus